### DISCUSSION TITLE
Export a function

### DISCUSSION THREAD

**Discussion by LSAXYZ321** [Qalculate/qalculate-gtk#469] [General]
Hi is there any way to export one or more functions from Qalculate, from one pc to another pc ?

**Comment by hanna-kn** [comment]
Copy `functions.xml` (from/to the user data folder/directory, e.g. `~/.local/qalculate/`, `C:\Users\[USERNAME]\AppData\Local\Qalculate\definitions`, or `qalculate\user\definitions`, depending on operating system and installation variant).

Edit the file to if you do not want to copy all user functions.

Rename the file (any name is allowed as long as the file is placed in the correct folder) if you do not wish to replace any user functions on the other pc.

I will consider adding an option to export one or multiple functions, units, and/or variables. There is already an option in qalculate-gtk to import definitions (which copies the selected file to the appropriate directory). In qalculate-qt there is an option the save/load "workspaces", which includes temporary variables and functions.

**Reply by LSAXYZ321** [reply]
Thanks a lot, that was helpful and an easy solution
